README.md

```diff
@@ -45,7 +45,7 @@ git clone https://github.com/phetsims/tandem.git
 git clone https://github.com/phetsims/twixt.git
 git clone https://github.com/phetsims/utterance-queue.git
 ```
-(2) Start an http-server
+(2) Start a http-server
```

Comment:
```suggestion
(2) Start a HTTP server
```
Why not do that? If you're fixing a grammatical error

Reply:
It seems there is a difference in grammar dependent on country of origin (see this [Quora thread](https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage#:~:text=Originally%20Answered%3A%20'A%20http) ).

We are trying to improve our open source processes here. We saw that this pull request was never addressed, and apologize for that. 

This file is a generated file, so if you do want to make the change please do so in chipper/templates/README-published.md. The regeneration happens overnight, so you do not have to worry about propagating across repos. Below is a patch, to help provide more guidance.

In order to submit the pull request, you will also need to sign this [document](https://powerforms.docusign.net/942abb00-7132-498e-94ce-e55344d32dd9?env=na2&acct=088d5d64-ef4d-40bb-acf2-480eabbf546d&accountId=088d5d64-ef4d-40bb-acf2-480eabbf546d). We are also happy to do the change ourselves if you'd prefer. 

```diff
Subject: [PATCH] update doc, https://github.com/phetsims/phet-core/issues/103
---
Index: templates/README-published.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/README-published.md b/templates/README-published.md
--- a/templates/README-published.md	(revision fd67663ce0105fdd4f306ec838ddc890422350af)
+++ b/templates/README-published.md	(date 1678218879327)
@@ -37,7 +37,7 @@
 (3) Change directory to chipper `cd ../chipper/`, then transpile the code to JavaScript by running `node js/scripts/transpile.js --watch`. This starts a file-watching process
 that will automatically transpile new or changed files.
 
-(4) In a new terminal/command prompt, start an http-server
+(4) In a new terminal/command prompt, start an HTTP server (like npm's [http-server](https://www.npmjs.com/package/http-server))
 
 (5) Open in the browser: `[http://localhost/{{REPOSITORY}}/{{REPOSITORY}}_en.html](http://localhost/%7B%7BREPOSITORY%7D%7D/%7B%7BREPOSITORY%7D%7D_en.html)` (You will probably need to modify this URL based on your HTTP port and relative path.)
```

Thanks for your contribution! :-)